tests/test_kernelbuilder.py

```diff
@@ -180,10 +179,9 @@ def test_getrelease(self):
             result = self.kbuilder.getrelease()
             self.assertEqual(kernel_version.strip(), result)
 
-    def test_getrelease_regex_fail(self):
-        """Ensure get_release() fails if the regex doesn't match"""
-        mock_prepare = mock.patch("skt.kernelbuilder.KernelBuilder.prepare")
-
+    @mock.patch("skt.kernelbuilder.KernelBuilder.prepare_kernel_config")
+    def test_getrelease_regex_fail(self, mock_prepare):
+        """Ensure get_release() fails if the regex doesn't match."""
```

Comment:
These fixes need to go in the same commit which introduces the breaking change, so there is no commit where tests are broken. Please move appropriately.

Reply:
Fixed.